acceptance/setup/git/pre-suite/000_EnvSetup.rb

```diff
@@ -12,6 +12,7 @@
     'git',
     'ruby',
     'rubygem-json',
+    'rubygem-io-console'
```

Comment:
@jcoconnor nevermind on adding it to images - that doesn't make sense given images don't have Ruby and we're also installing Ruby as part of the setup in https://github.com/puppetlabs/puppet/blob/9ccdd7e690033ec93e87b9a8e2ab853f0c21bd7d/acceptance/setup/git/pre-suite/000_EnvSetup.rb#L10-L40